helix-term/src/commands.rs

```diff
@@ -1291,7 +1291,8 @@ fn global_search(cx: &mut Context) {
 
     cx.push_layer(Box::new(prompt));
 
-    let root = find_root(None).unwrap_or_else(|| PathBuf::from("./"));
+    let current_path = doc_mut!(cx.editor).path().cloned();
```

Comment:
Actually, this keeps the path relative to the working directory whereas other pickers use the project root. Can we keep the original behavior? Either that or other code paths should be updated to align the behaviors.